code/ATMOSPHERICS/pipe/construction.dm

```diff
@@ -436,7 +435,6 @@ var/list/heat_pipes = list(PIPE_HE_STRAIGHT, PIPE_HE_BENT, PIPE_JUNCTION)
 
 		if(PIPE_JUNCTION)
 			P=new /obj/machinery/atmospherics/pipe/simple/heat_exchanging/junction( src.loc )
-			investigation_log(I_ATMOS,"was created by [user]/([user.ckey]) at [formatJumpTo(loc)].")
```

Comment:
And these lines got moved to further down
This way all pipe constructions are logged, and they're also only logged if they actually succeed in building